tests/integration/targets/git_config/tasks/set_value_with_tilde.yml

```diff
@@ -27,10 +27,14 @@
     scope: global
   register: get_result
 
+- debug: var=set_result1
+- debug: var=set_result2
+- debug: var=get_result
```

Comment:
All these `debug` tasks necessary (also in the other files)? By default integration tests run with `-v`, so the results are already printed (though with less verbosity).

Reply:
Thx. Done